controllers/packagebundlecontroller_controller.go

```diff
@@ -114,12 +114,22 @@ func (r *PackageBundleControllerReconciler) Reconcile(ctx context.Context, req c
 		return withoutRequeue(result), nil
 	}
 
+	if pbc.Status.State != api.BundleControllerStateActive {
+		pbc.Status.State = api.BundleControllerStateActive
+		r.Log.V(6).Info("update", "PackageBundleController", pbc.Name, "state", pbc.Status.State)
+		err = r.Client.Status().Update(ctx, pbc, &client.UpdateOptions{})
+		if err != nil {
+			return result, fmt.Errorf("updating status: %s", err)
+		}
+		return result, nil
+	}
+
 	latestBundle, err := r.bundleManager.LatestBundle(ctx, pbc.Spec.Source.BaseRef())
 	if err != nil {
 		r.Log.Error(err, "Unable to get latest bundle")
 		if pbc.Status.State != api.BundleControllerStateDisconnected {
```

Comment:
Is this check still necessary? Shouldn't pbc.Status.State be active for all pbc that can reach this step?